Examples/TicTacToe/tic-tac-toe/Tests/GameSwiftUITests/GameSwiftUITests.swift

```diff
@@ -5,83 +5,62 @@ import XCTest
 @testable import GameSwiftUI
 
 class GameSwiftUITests: XCTestCase {
-  let store = TestStore(
+  let store = Store(
     initialState: GameState(
       oPlayerName: "Blob Jr.",
       xPlayerName: "Blob Sr."
     ),
     reducer: gameReducer,
     environment: GameEnvironment()
   )
-  .scope(state: GameView.ViewState.init)
+  var viewStore: ViewStore<GameView.ViewState, GameAction> {
+    ViewStore(store.scope(state: GameView.ViewState.init))
+  }
 
   func testFlow_Winner_Quit() {
-    self.store.send(.cellTapped(row: 0, column: 0)) {
-      $0.board[0][0] = "❌"
-      $0.title = "Blob Jr., place your ⭕️"
-    }
```

Comment:
I think we could also test this kinda stuff more exhaustively with a generalized helper:

```swift
XCTAssertDifferenceStartingFrom(self.viewStore.state) {
  self.store.send(.cellTapped(row: 0, column: 0))
} assert: {
  $0.board[0][0] = "❌"
  $0.title = "Blob Jr., place your ⭕️"
}
```

Not sure what repo we'd put such a helper in, though.

Reply:
> Not sure what repo we'd put such a helper in, though.

Duh 😂 https://github.com/pointfreeco/swift-custom-dump/pull/51